resources/translations/en.edn

```diff
@@ -17,7 +17,7 @@
             :cancel "Cancel"
             :close "Close application"
             :close-intro [:div.intro [:p "Note that if the application is closed, it cannot be opened again."]]
-            :comment "Comment"
+            :comment "Review"
```

Comment:
Hmm, maybe use "katselmoi" or "arvioi" or something similar in the Finnish translations.

Reply:
Oops, forgot all about finnish...